machine_manager/README.rst

```diff
@@ -0,0 +1,27 @@
+Machine Manager
```

Comment:
Maybe it could be used with more generic "equipment" but there are some fields that doesn't fit so well. And for vehicles we have an official module ```fleet``` already.